src/styles/blocks/catalog.scss

```
@@ -17,9 +18,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all $smooth-animate ease-in-out;
```

don't use it like that - It's too hard for the browser
Check this
https://github.com/mate-academy/layout_catalog_hovers/blob/master/checklist.md

```suggestion
    transition: all $smooth-animate ease-in-out;
```